.github/scripts/validateActionsAndWorkflows.sh

```diff
@@ -94,4 +100,43 @@ fi
 # Cleanup after ourselves and delete actionlint
 rm -rf ./actionlint
 
+###############################################################################
+#                      Check for unsafe action references                     #
+###############################################################################
+title 'Checking for mutable action references...'
+find . -type f \( -name "*.yml" -o -name "*.yaml" \) -print0 \
+  | xargs -0 grep --no-filename "uses:" \
+  | sed 's/\- uses:/uses:/g' \
+  | tr '"' ' ' \
+  | awk '{print $2}' \
+  | sed 's/\r//g' \
+  | grep -vE '\.github\/' \
+  | sort \
+  | uniq \
+  > /tmp/actionUsages.txt
+
+info 'All action usages...'
+cat /tmp/actionUsages.txt
+
+TRUSTED_ORGS=(
+  '^Expensify/'
+)
+grep -vE "$(IFS='|'; echo "${TRUSTED_ORGS[*]}")" /tmp/actionUsages.txt > /tmp/untrustedActionUsages.txt
+echo ''
```

Comment:
NAB empty string is not necessary

```suggestion
echo
```

Reply:
TIL